source/crud/tests/unified/bulkWrite-updateOne-sort.yml

```diff
@@ -0,0 +1,125 @@
+description: BulkWrite updateOne-sort
+schemaVersion: "1.0"
+createEntities:
+  - client:
+      id: client0
+      observeEvents:
+        - commandStartedEvent
+        - commandSucceededEvent
+  - database:
+      id: database0
+      client: client0
+      databaseName: crud-tests
+  - collection:
+      id: collection0
+      database: database0
+      collectionName: coll0
+initialData:
+  - collectionName: coll0
+    databaseName: crud-tests
+    documents:
+      - _id: 1
+        x: 11
+      - _id: 2
+        x: 22
+      - _id: 3
+        x: 33
+tests:
+  - description: BulkWrite updateOne with sort option
+    runOnRequirements:
+      - minServerVersion: "8.0"
+    operations:
+      - object: collection0
+        name: bulkWrite
+        arguments:
+          requests:
+            - updateOne:
+                filter:
+                  _id:
+                    $gt: 1
+                sort:
+                  _id: -1
+                update:
+                  - $set:
+                      x: 1
+    expectEvents:
+      - client: client0
+        events:
+          - commandStartedEvent:
+              command:
+                update: coll0
+                updates:
+                  - q:
+                      _id:
+                        $gt: 1
+                    u:
+                      - $set:
+                          x: 1
+                    sort:
+                      _id: -1
+                    multi:
+                      $$unsetOrMatches: false
+                    upsert:
+                      $$unsetOrMatches: false
+          - commandSucceededEvent:
+              reply: { ok: 1, n: 1 }
+              commandName: update
+    outcome:
+      - collectionName: coll0
+        databaseName: crud-tests
+        documents:
+          - _id: 1
+            x: 11
+          - _id: 2
+            x: 22
+          - _id: 3
+            x: 1
+  - description: BulkWrite updateOne with sort option unsupported (server-side error)
+    runOnRequirements:
+      - maxServerVersion: "7.99"
+    operations:
+      - object: collection0
+        name: bulkWrite
+        arguments:
+          requests:
+            - updateOne:
+                filter:
+                  _id:
+                    $gt: 1
+                sort:
+                  _id: -1
+                update:
+                  - $set:
+                      x: 1
+        expectError:
+          errorContains: "'update.updates.sort' is an unknown field"
```

Comment:
Spec tests generally don't assert the contents of server-side error messages since they can be inconsistent across server versions or otherwise subject to change without notice. `errorContains` is more commonly used to assert client-side messages, where a spec enforces a particular format.

Consider using an error code or just relying on `isClientError: false`.